dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3JobTask.java

```diff
@@ -89,7 +89,7 @@ void hostNotAvailable() {
         final String msg = resourceBundle.getString("host") + SPACE + ds3Client.getConnectionDetails().getEndpoint() + resourceBundle.getString("unreachable");
         ErrorUtils.dumpTheStack(msg);
         loggingService.logMessage(resourceBundle.getString("unableToReachNetwork"), LogType.ERROR);
-        new LazyAlert(resourceBundle).error(msg);
+        new LazyAlert(resourceBundle).errorRaw(resourceBundle.getString(msg));
```

Comment:
If we're replacing the LazyAlert with the AlertService, this class should have the AlertService injected instead of the LazyAlert being created here.